scylla-persistence/src/main/java/com/netflix/conductor/scylla/config/ScyllaProperties.java

```diff
@@ -74,6 +74,25 @@ public class ScyllaProperties {
     @DurationUnit(ChronoUnit.SECONDS)
     private Duration eventExecutionPersistenceTtl = Duration.ZERO;
 
+    private int ttlShardIdCache = 10;
```

Comment:
- Can you please add the unit in the comments after ';'
for both the constants
- This could be static final as well

Reply:
putting it here to make it configurable